docs/content/developer/cryptography/transaction-auth/multisig.mdx

```diff
@@ -61,27 +61,33 @@ The response resembles the following, but displays actual addresses, keys, and p
-│ │ flag            │  0                                                                   │ │
-│ │ peerId          │  <PEER-ID>                                                           │ │
-│ ╰─────────────────┴──────────────────────────────────────────────────────────────────────╯ │
-│ ╭─────────────────┬──────────────────────────────────────────────────────────────────────╮ │
-│ │ iotaAddress      │  <IOTA-ADDRESS>                                                       │ │
-│ │ publicBase64Key │  <PUBLIC-KEY>                                                        │ │
-│ │ keyScheme       │  secp256k1                                                           │ │
-│ │ flag            │  0                                                                   │ │
-│ │ peerId          │  <PEER-ID>                                                           │ │
-│ ╰─────────────────┴──────────────────────────────────────────────────────────────────────╯ │
-│ ╭─────────────────┬──────────────────────────────────────────────────────────────────────╮ │
-│ │ iotaAddress      │  <IOTA-ADDRESS>                                                       │ │
-│ │ publicBase64Key │  <PUBLIC-KEY>                                                        │ │
-│ │ keyScheme       │  secp256r1                                                           │ │
-│ │ flag            │  0                                                                   │ │
-│ │ peerId          │  <PEER-ID>                                                           │ │
-│ ╰─────────────────┴──────────────────────────────────────────────────────────────────────╯ │
+│ ╭─────────────────────────┬──────────────────────────────────────────────────────────────╮ │
+│ │ alias                   │  <ALIAS>                                                     │ │
+│ │ iotaAddress             │  <IOTA-ADDRESS>                                              │ │
+│ │ publicBase64Key         │  <PUBLIC-KEY>                                                │ │
+│ │ publicBase64KeyWithFlag │  <PUBLIC-KEY-WITH-FLAG>                                      │ │
+│ │ keyScheme               │  ed25519                                                     │ │
+│ │ flag                    │  0                                                           │ │
+│ │ peerId                  │  <PEER-ID>                                                   │ │
+│ ╰─────────────────────────┴──────────────────────────────────────────────────────────────╯ │
+│ ╭─────────────────────────┬──────────────────────────────────────────────────────────────╮ │
+│ │ alias                   │  <ALIAS>                                                     │ │
+│ │ iotaAddress             │  <IOTA-ADDRESS>                                              │ │
+│ │ publicBase64Key         │  <PUBLIC-KEY>                                                │ │
+│ │ publicBase64KeyWithFlag │  <PUBLIC-KEY-WITH-FLAG>                                     │ │
```

Comment:
I think there needs to be additional spacing now 😆

Reply:
😆 Fixed